### DISCUSSION TITLE
Separate Aggregation Server from the Logger

### DISCUSSION THREAD

**Discussion by gibbsjj** [mbridak/FieldDayLogger#47] [Ideas]
I'd like to run the server on a Pi Zero W that is not running any X11 desktop. It would be awesome if the server could be run without requiring PyQt5!

**Comment by mbridak** [comment]
Hi, the [aggregtion server](https://github.com/mbridak/fdserver) doesn't run any x11/qt code. It uses ncurses and runs in a terminal window.
It does require a 80 x 24 character display.

**Comment by gibbsjj** [comment]
Somehow I missed that - thanks! This is such an awesome project. Can't wait for FD '25! 73!
